ios/FluentUI/Vnext/Persona/PersonaView.swift

```diff
@@ -0,0 +1,185 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc public protocol MSFPersonaViewState: MSFAvatarState {
+    var titleTrailingAccessoryView: UIView? { get set }
+    var subtitleTrailingAccessoryView: UIView? { get set }
+    var onTapAction: (() -> Void)? { get set }
+}
+
+class MSFPersonaViewStateImpl: MSFListCellState, MSFPersonaViewState {
+    init(avatarState: MSFAvatarState) {
+        self.avatarState = avatarState
+    }
+
+    var image: UIImage? {
+        get {
+            return avatarState.image
+        }
+
+        set {
+            avatarState.image = newValue
+        }
+    }
+
+    var primaryText: String? {
+        get {
+            return avatarState.primaryText
+        }
+
+        set {
+            avatarState.primaryText = newValue
+            title = newValue ?? ""
+        }
+    }
+
+    var secondaryText: String? {
+        get {
+            return avatarState.secondaryText
+        }
+
+        set {
+            avatarState.secondaryText = newValue
+            subtitle = newValue ?? ""
+        }
+    }
+
+    var ringColor: UIColor? {
+        get {
+            return avatarState.ringColor
+        }
+
+        set {
+            avatarState.ringColor = newValue
+        }
+    }
+
+    var backgroundColor: UIColor? {
+        get {
+            return avatarState.backgroundColor
+        }
+
+        set {
+            avatarState.backgroundColor = newValue
+        }
+    }
+
+    var foregroundColor: UIColor? {
+        get {
+            return avatarState.foregroundColor
+        }
+
+        set {
+            avatarState.foregroundColor = newValue
+        }
+    }
+
+    var presence: MSFAvatarPresence {
+        get {
+            return avatarState.presence
+        }
+
+        set {
+            avatarState.presence = newValue
+        }
+    }
+
+    var isRingVisible: Bool {
+        get {
+            return avatarState.isRingVisible
+        }
+
+        set {
+            avatarState.isRingVisible = newValue
+        }
+    }
+
+    var isTransparent: Bool {
+        get {
+            return avatarState.isTransparent
+        }
+
+        set {
+            avatarState.isTransparent = newValue
+        }
+    }
+
+    var isOutOfOffice: Bool {
+        get {
+            return avatarState.isOutOfOffice
+        }
+
+        set {
+            avatarState.isOutOfOffice = newValue
+        }
+    }
+
+    private var avatarState: MSFAvatarState
+}
+
+struct PersonaView: View {
```

Comment:
Should be public for the SwiftUI consumption.
Also, shouldn't this be named **MSFPersonaView**?